src/lightning/pytorch/loggers/mlflow.py

```diff
@@ -138,7 +143,7 @@ def __init__(
         self._checkpoint_callback: Optional[ModelCheckpoint] = None
         self._prefix = prefix
         self._artifact_location = artifact_location
-
+        self._kwargs = {} if synchronous is None else {"synchronous": synchronous}
```

Comment:
I suggest to make this name specific for how it's used.
```suggestion
        self._log_batch_kwargs = {} if synchronous is None else {"synchronous": synchronous}
```